app/src/main/java/com/srivatsa/tipcalculator/MainActivity.kt

```diff
@@ -19,58 +18,16 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        seekBarTip.progress=INITIAL_TIP_PERCENT
-        tvTipPercent.text="$INITIAL_TIP_PERCENT%"
-        updateTipDescription(INITIAL_TIP_PERCENT)
-
-
-        seekBarTip.setOnSeekBarChangeListener(object: SeekBar.OnSeekBarChangeListener {
-            override fun onProgressChanged(p0: SeekBar?, p1: Int, p2: Boolean)
-            {
-                Log.i(TAG, "onProgressChanged $p1")
-                tvTipPercent.text="$p1%"
-                computeTipAndTotal()
-                updateTipDescription(p1)
-            }
-
-            override fun onStartTrackingTouch(p0: SeekBar?) {}
-
-            override fun onStopTrackingTouch(p0: SeekBar?) {}
-
-        })
-
         etBase.addTextChangedListener(object:TextWatcher{
             override fun afterTextChanged(s: Editable?) {
                 Log.i(TAG,"afterTextChanged $s")
                 computeTipAndTotal()
-
             }
-
             override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {}
-
             override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {}
         })
     }
 
-    private fun updateTipDescription(tipPercent: Int) {
-
-        val tipDescription: String
-        when(tipPercent){
-            in 0..9 -> tipDescription="Poor"
-            in 10..14 -> tipDescription="Acceptable"
-            in 15..19 -> tipDescription="Great"
-            else -> tipDescription="Amazing"
-
-        }
-        tvTipDescription.text=tipDescription
-        val color=ArgbEvaluator().evaluate(tipPercent.toFloat()/seekBarTip.max,
-            ContextCompat.getColor(this,R.color.worstTip),
-            ContextCompat.getColor(this, R.color.BestTip)
-        )as Int
-        tvTipDescription.setTextColor(color)
-
-    }
-
     private fun computeTipAndTotal() {
```

Comment:
It's a good practice to write functions with parameters and return type.
Something like below would be nice little functions that computes and returns data and will not change anything out of its scope.

`private fun computeTipAndTotal(baseAmount : Int, tipPercentage : Int) : Pair<Int,Int> {
        val tipAmount=baseAmount * tipPercentage /100
        val totalAmount = baseAmount + tipAmount
        return Pair(totalAmount,tipAmount)
    }`